app/src/components/buttons/filterButton/filterContent/filterInput/filterInput.jsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import { Dropdown, FieldText } from '@reportportal/ui-kit';
+import PropTypes from 'prop-types';
+import styles from './filterInput.scss';
+
+const cx = classNames.bind(styles);
+
+export const FilterInput = ({ filter, setFilters }) => {
+  const { filterName, options, value, condition, placeholder, title, withField } = filter;
+
+  return (
+    <div className={cx('filter-item')}>
+      <span className={cx('label')}>{title}</span>
+      <div className={cx('container')}>
+        <Dropdown
```

Comment:
I'm not sure what is the reason, but looks like the active/focus state is broken for dropdown options right now (it's too large).
![image](https://github.com/user-attachments/assets/e2e9b1b1-26ae-4386-b67b-acadda730a53)

Reply:
This problem is on the ui-kit side

Reply:
Pls, notify QA team about that issue.